src/main/scala/kamon/prometheus/PrometheusReporter.scala

```diff
@@ -97,12 +97,21 @@ class PrometheusReporter extends MetricReporter {
       embeddedServerPort = prometheusConfig.getInt("embedded-server.port"),
       defaultBuckets = prometheusConfig.getDoubleList("buckets.default-buckets").asScala,
       timeBuckets = prometheusConfig.getDoubleList("buckets.time-buckets").asScala,
-      informationBuckets = prometheusConfig.getDoubleList("buckets.information-buckets").asScala
+      informationBuckets = prometheusConfig.getDoubleList("buckets.information-buckets").asScala,
+      customBuckets = getMappedDoubleList("buckets.custom", prometheusConfig)
     )
   }
+
+  private def getMappedDoubleList(key: String, conf: Config) = {
+    val cnf = conf.getConfig(key)
```

Comment:
There are a few tools in Kamon that make this a bit simpler: https://github.com/kamon-io/Kamon/blob/master/kamon-core/src/main/scala/kamon/package.scala#L87, it would be great if you could change it to be consistent with that. Ping me if you need help with it!

Reply:
will have a look next week

Reply:
when I apply the .configurations on my buckets config, I get

> application.conf @ file:.../application.conf: 51: akka_actor_processing_time_seconds has type LIST rather than OBJECT

In contrast to the core example, where it gets really config blocks, the Config objects in my case are the lists of doubles